run-tox-env.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       TMPDIR: $(Agent.TempDirectory)
       PIP_NO_WARN_SCRIPT_LOCATION: '0'
       PIP_DISABLE_PIP_VERSION_CHECK: '1'
+      PIP_CONSTRAINT: ${{ coalesce(job.value.PIP_CONSTRAINT, '') }}
```

Comment:
wouldn't be a more general solution to put here:

```
    - ${{ if job.value.variables }}:
      ${{ job.value.variables }}
```

Reply:
Lets try it, YAML programming is not as fun as Python.